.circleci/config.yml

```diff
@@ -35,6 +35,8 @@ jobs:
       - run:
           name: Run integration tests
           command: ./scripts/circleci/run_automation_test.sh -t integration
+          environment:
+          API_KEY: ${API_KEY}
```

Comment:
I don't think this change is required, it's just setting the variable to itself

Reply:
ok.. I'll remove this and let circle ci run.. thanks